### ISSUE TITLE
你好,关于风格迁移这块有一点疑惑

### ISSUE THREAD

**Issue by Apokar** [PaddlePaddle/PaddleFormers#1504]
[参照教程](https://github.com/PaddlePaddle/PaddleHub/blob/release/v2.1/demo/style_transfer/README.md)
finetune的第四部很慢,GPU没有参与进来 这样正常吗
代码用readme里的没有改动,添加了两句import 

- 版本、环境信息
1）
paddle2onnx                   0.6
paddlehub                     2.1.0
paddlenlp                     2.0.3
paddlepaddle-gpu              2.1.0

2）系统环境：
Ubuntu 18 
python 3.7
- 复现信息：
step4: 选择优化策略和运行配置 之后的日志信息

[2021-06-24 20:28:15,045] [ WARNING] - PaddleHub model checkpoint not found, start from scratch...
load pretrained vgg16 checkpoint success
[2021-06-24 20:35:08,535] [   TRAIN] - Epoch=1/101, Step=10/501 loss=379440.7109 content gap=205422.5781 style gap=174018.1094 lr=0.000100 step/sec=0.02 | ETA 581:11:32

**Comment by Apokar** [comment]
没事了 确实是没有调用gpu  
trainer = Trainer(model, optimizer, checkpoint_dir='test_style_ckpt',**use_gpu=True)** 就正常了
